app/chef/components/VideoPlayer.tsx

```
@@ -0,0 +1,686 @@
+                <button
+                  onClick={() => handlePopupAction('close')}
+                  className="absolute right-3 top-3 bg-transparent border-none p-0 cursor-pointer"
+                >
+                  <XMarkIcon className="w-6 h-6 text-[#F0725C] hover:text-[#FE3511] transition-colors" />
+                </button>
+              </div>
+            </div>
+          </div>
+        )}
+
+        {countdown !== null && isNaN(countdown) && typePopup && (
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)